_data/projects.yml

```diff
@@ -47,6 +47,22 @@
   img_url: assets/img/nodejs-sdk-project.webp
   stage: Generally Available
   dedication: Core
+- name: Jobs
+  description: Provides APIs to handle jobs access. Use the z/OSMF directly as it provides richer functionality.
+  github_url: https://github.com/zowe/jobs
+  slack_url: https://github.com/zowe/api-layer
```

Comment:
How about a direct Slack link? (https://openmainframeproject.slack.com/archives/CC5UUL005)

Reply:
Thanks for catching that. Fixed now